### PR TITLE
Vcon 92 chat message service

### PR DESCRIPTION
**Pull request by nibivi77**
### Description
Added the [ChatService](https://decdvirtualconcierge.atlassian.net/jira/software/projects/VCON/boards/2?selectedIssue=VCON-92) with a fake JSON file in our public folder, under data.

## Notes

- We need to figure how to handle the dates. for now they are stored in english instead of time. This will affect our sorting logic.
- I am not sure if this PR is using the best practices so please be vigilant when reviewing this one.

### CODE REVIEW

File: src/store/modules/chatMessages.js [hunk 1]
```diff
@@ -1,123 +1,49 @@
...
-        return -1;
-      }
-      if (a.receivedTime < b.receivedTime) {
-        return 1;
-      }
-      return 0;
-    });
-    return conversation;
+    if (state.chatMessages.length !== 0) {
+      const conversation = [...state.chatMessages[0].messages].sort((a, b) => {
```
Comment (TishamIslam) [inline src/store/modules/chatMessages.js:13]: You could use state.chatMessages[0].messages here, since messages is an array.

Comment (DaceyTom2) [reply src/store/modules/chatMessages.js:13]: To add onto this we can specify which chatMessages to get now from inbox/selectedInboxItemId. We may need to coordinate this change

**Review by DaceyTom2 — APPROVED** [review]
Looks good and keep the change you filthy animal

File: src/store/modules/chatMessages.js (hunk 1 above)
Comment (nibivi77) [reply src/store/modules/chatMessages.js:13]: Taking a look now.

Comment (nibivi77) [reply src/store/modules/chatMessages.js:13]: I actually can't do what is suggested because that would change the state. A getter cannot mutate the state.

Comment (TishamIslam) [reply src/store/modules/chatMessages.js:13]: Ohh, that makes sense. In that case, could you add a comment to clarify that ? It's not very obvious at first glance that this is a quick way to get a copy of the array.

Comment (nibivi77) [reply src/store/modules/chatMessages.js:13]: Sure!

**Pull request merged**